### PR TITLE
add info about Cloud Windows Hello for Business deployment model

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8423

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ ms.reviewer:
 
 - Windows 10
 - Certificate trust deployments
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud, Hybrid and On-premises Windows Hello for Business deployments
 - Azure AD joined, Hybrid Azure AD joined, and Enterprise joined devices
 - Certificate trust deployments
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:28]: Duplicate of line 25, redundant line.
```suggestion
```

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ ms.reviewer:
 
 - Windows 10
 - Certificate trust deployments
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:25]: Suggest removing this line instead, to keep the same order as in the Requirements section.
```suggestion
```
(duplicate of line 28)

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 3]
```diff
@@ -22,8 +22,7 @@ ms.reviewer:
 **Requirements**
 
 - Windows 10
-- Certificate trust deployments
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud, Hybrid and On-premises Windows Hello for Business deployments
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:25]:
```suggestion
- Cloud, Hybrid, and On-premises Windows Hello for Business deployments
```

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 4]
```diff
@@ -35,7 +34,7 @@ Microsoft continues to investigate supporting using keys trust for supplied cred
 
 **Requirements**
 
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud, Hybrid and On-premises Windows Hello for Business deployments
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:37]:
```suggestion
- Cloud, Hybrid, and On-premises Windows Hello for Business deployments
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - two minor edits.

**Comment by e0i** [comment]
@mapalko Can you take a second look at this PR? The copy editing suggestions have been applied. Thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@VLG17 - Please take another look. The second edit has not been applied.

**Review by illfated — APPROVED** [review]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 5]
```diff
@@ -22,8 +22,7 @@ ms.reviewer:
 **Requirements**
 
 - Windows 10
-- Certificate trust deployments
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud, Hybrid, and On-premises Windows Hello for Business deployments
 - Azure AD joined, Hybrid Azure AD joined, and Enterprise joined devices
 - Certificate trust deployments
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:27]: The duplicate mention of certificate trust should be removed from the end of this list of requirements.

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 6]
```diff
@@ -35,7 +34,7 @@ Microsoft continues to investigate supporting using keys trust for supplied cred
 
 **Requirements**
 
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud, Hybrid, and On-premises Windows Hello for Business deployments
 - Azure AD joined, Hybrid Azure AD joined, and Enterprise joined devices
 - Certificate trust deployments
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:39]: The duplicate mention of certificate trust should also be removed from this mention.

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 7]
```diff
@@ -22,8 +22,7 @@ ms.reviewer:
 **Requirements**
 
 - Windows 10
-- Certificate trust deployments
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud, Hybrid, and On-premises Windows Hello for Business deployments
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:25]:
```suggestion
- Cloud only, Hybrid, and On-premises only  Windows Hello for Business deployments
```

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 8]
```diff
@@ -35,7 +34,7 @@ Microsoft continues to investigate supporting using keys trust for supplied cred
 
 **Requirements**
 
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud, Hybrid, and On-premises Windows Hello for Business deployments
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:37]:
```suggestion
- Cloud only, Hybrid, and On-premises only Windows Hello for Business deployments
```

**Comment by mapalko** [comment]
The external comment is somewhat misleading. For cloud only deployments, a certificate is not needed for RDP. The limitation is that you can only RDP between Azure AD Joined hosts. Certificates are an AD concept and are needed in hybrid or on-prem scenarios for RDP. Adding a certificate in a cloud only environment does not really make sense since there is no authority that can be used to issue or validate the cert. I am not sure of the best way to represent that in this doc though.

**Review by mapalko — CHANGES_REQUESTED** [review]
See above change requests.

**Review by VLG17 — COMMENTED** [review]
as advised by mapalko

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md [hunk 9]
```diff
@@ -35,7 +33,7 @@ Microsoft continues to investigate supporting using keys trust for supplied cred
 
 **Requirements**
 
-- Hybrid and On-premises Windows Hello for Business deployments
+- Cloud only, Hybrid, and On-premises only Windows Hello for Business deployments
 - Azure AD joined, Hybrid Azure AD joined, and Enterprise joined devices
 - Certificate trust deployments
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:38]:
```suggestion
```

File: windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md (hunk 6 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-feature-remote-desktop.md:38]: @VLG17 : See my removal comment further down in the comment section.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Review by mapalko — APPROVED** [review]

**Pull request merged**